hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/erasurecode/StripedReconstructor.java

```diff
@@ -288,6 +292,10 @@ RawErasureDecoder getDecoder() {
     return decoder;
   }
 
+  int getIndicesNum(){
+    return liveIndiceNum;
+  }
+
```

Comment:
@Kidd53685368 Thanks for your PR.
How about using `BitSet#cardinality` instead of calculating `liveIndiceNum`?
```suggestion
  int getNumLiveBlocks(){
    return liveBitSet.cardinality();
  }

```

Reply:
Thanks, this is absolutely a better substitution.